modules/nf-core/samtools/sormadup/meta.yml

```diff
@@ -0,0 +1,110 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "samtools_sormadup"
+description: Collate/Fixmate/Sort/Markdup SAM/BAM/CRAM file
+keywords:
+  - cat
+  - collate
+  - fixmate
+  - sort
+  - markduplicates
+  - bam
+  - sam
+  - cram
```

Comment:
I'd add another keyword for denoting such a module

Reply:
suggestions?

Reply:
Suggestions: 
- `multi-tool` // My pick
-  `optimised`
- `enhanced`
- `performant`

Reply:
`multi-tool` it is!

Reply:
+1 for `multi-tool`